expire.js

```diff
@@ -53,17 +53,15 @@ utils.init_db().then(async () => {
         (async function(index) {
             let user = users[index];
             console.log('User: ' + user.uid + ' has expired');
-            let msg_activ = 'User ' + user.uid + ' has expired, updating account';
-            let msg_activ_html = msg_activ;
-            let mailOptions = {
-                origin: MAIL_CONFIG.origin, // sender address
-                destinations: [CONFIG.general.support], // list of receivers
-                subject: CONFIG.general.name + ' account expiration: ' + user.uid, // Subject line
-                message: msg_activ, // plaintext body
-                html_message: msg_activ_html // html body
-            };
             try {
-                await notif.sendUser(mailOptions);
+                await utils.send_notif_mail({
+                    'name': 'expired',
+                    destinations: [CONFIG.general.support],
+                    subject: 'account expire: ' + user.uid
```

Comment:
please keep "expiration"

Reply:
if I use "expiration", I will have to rename the current "expiration" as "expiration_warning" (there is 2 mail for expiration, one before and one after)

Reply:
I meant "account expiration" or "account expired" for subject

Reply:
https://github.com/genouest/genouestaccountmanager/pull/166/commits/bca560ac99cd7eeb97dc32146e82c0fca13232d0